infrastructure/tooling/src/publish/tasks.js

```diff
@@ -510,21 +509,23 @@ module.exports = ({tasks, cmdOptions, credentials, baseDir, logsDir}) => {
         .map(name => ({name, contentType: 'application/octet-stream'}));
       for (let {name, contentType} of files) {
         utils.status({message: `Upload Release asset ${name}`});
-        const file = await readFile(path.join(artifactsDir, name));
+        const data = await readFile(path.join(artifactsDir, name));
 
         /* Artifact uploads to GitHub seem to fail.. a lot.  So we retry each
          * one a few times with some delay, in hopes of getting lucky.  */
         let retries = 5;
         while (retries-- > 0) {
           try {
             await octokit.repos.uploadReleaseAsset({
-              url: upload_url,
+              owner: 'taskcluster',
+              repo: 'taskcluster',
```

Comment:
do we hardcode that we are `taskcluster/taskcluster` anywhere else?

Reply:
A few lines earlier, among other places :)